examples/todo-list/src/AddTodoTask.js

```diff
@@ -0,0 +1,35 @@
+import { useState } from "react";
+import { useStore } from "./store";
+import { FcTodoList } from "react-icons/fc";
+
+export default function AddTodoTask() {
+  const [counter, setCounter] = useState(0);
```

Comment:
I feel this counter inconsistent, navegating to another page + come back (unmount + mount) is going to repeat id's...

Reply:
having in mind this todo app is not growing and there is no plan to change page or unmount the component, I think we could leave as is